### ISSUE TITLE
Properly configure test and stage handle server

### ISSUE THREAD

**Issue by mfenner** [datacite/mastino#90]

**Comment by richardhallett** [comment]
@mfenner Can you expand upon what the work here is? Are there distinct setups for them? are there specific problems? Or is it the case I need to do some testing to ensure their setup?

**Comment by mfenner** [comment]
Do you need to do anything special (one-time setup) when starting the docker container? The `stage` system (the original container) seems to be running fine, but not the new `test` one.

We also may need to tweek `lupo` and potentially `bracco` as I think we have hard-wired the handle system in some places.

**Comment by richardhallett** [comment]
Both are working, the outstanding work will be to move stage to it's own handle database.
This is dependent on the work of copying the DOI database, see github.com/datacite/mastino#88

Once this work is complete I think the best idea is to do an export of the handle database from test and copy this to stage. mysqldump etc.
There maybe additional work to configure prefixes that are available.

**Comment by richardhallett** [comment]
I've looked at this a bit and given it some thought. I actually not sure its worth doing. if the whole point is that we use stage for ourselves then it will be only us trying to make updates to prefixes that we control, probably on datacite.X repositories. In which case the chance of conflict is minimal I think. Additionally it wont not work, what you might end up doing is changing the URL for the handle to a different value that is stored in the opposite database.

Another way also to handle this would be to control prefixes better, if you limited prefixes to set ones between the two, bit like to different RA, then you dont need extra DB.

I've updated the internal wiki to document the usage of prefixes on the handle server in that they should be unique between the systems.

I think this may all want revisiting at a later date, but for now I think we can close this issue.
